fetch/main.ts

```diff
@@ -0,0 +1,22 @@
+interface Pokemon {
+  name: string;
+  height: number;
+  weight: number;
+}
+
+async function fetchData(): Promise<void> {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+
+    if (!response.ok) {
+      throw new Error(`HTTP error! Status: ${response.status}`);
+    }
+
+    const data = (await response.json()) as Pokemon;
```

Comment:
You have a Pokemon interface here but you are sending a request to the `jsonplaceholder.typicode.com/users`. These users aren't Pokemon. I would either say that you should change your interface to `User` and update the properties with users, or update your `fetch` url to send a request to the `pokeAPI`